drivers/SmartThings/sonos/src/api/sonos_connection.lua

```diff
@@ -248,7 +256,9 @@ end
 --- Whether or not the connection has all of the live websocket connections it needs to function
 --- @return boolean
 function SonosConnection:is_running()
-  return self:self_running() and self:coordinator_running()
+  local self_running = self:self_running()
+  local coord_running = self:coordinator_running()
+  return  self_running and coord_running
```

Comment:
what's different here?

Reply:
I refactored it for debugging and left it in. There's no functionality changes. It could possibly be a perf improvement with the way Lua looks for upvalues but nothing excited.